### PR TITLE
Feature/colors UI

### PR DESCRIPTION
**Pull request by ttnguyen15**

### CODE REVIEW

File: src/styles/global.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@import "/src/assets/fonts/inter.css";
+@import "/src/styles/ui-colors.css";
+
+:root {
+    --max-width: 100%;
+    --border-radius: 12px;
+    --font: Inter;
+}
+
+* {
```
Comment (SabinePfanner) [inline src/styles/global.css:10]: I would recommend to put the styling for * on top, as it weights higher than `:root`

File: src/styles/global.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+html,
```
Comment (SabinePfanner) [inline src/styles/global.css:16]: Question: Where do you use `html `class?

File: src/styles/global.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+body {
+    max-width: 100vw;
+    overflow-x: hidden;
```
Comment (doemser) [inline src/styles/global.css:19]: this rule should not be needed and if it is needed, we should find the root of this issue somewhere else in your css

File: styles.js [hunk 4]
```diff
@@ -1,14 +0,0 @@
-import { createGlobalStyle } from "styled-components";
-
-export default createGlobalStyle`
-  *,
-  *::before,
```
Comment (SabinePfanner) [inline styles.js:5]: Removing `*::before` and `*::after` might lead to some unexpected behaviour - just as a remark

**Review by SabinePfanner — CHANGES_REQUESTED** [review]
Looks good to me, just some small suggestions

File: src/styles/global.css (hunk 1 above)
Comment (miToe) [reply src/styles/global.css:10]: there is nothing higher than the root element!

Comment (doemser) [reply src/styles/global.css:10]: normally you put the * to the top as it is has the lowest weight

File: src/styles/global.css (hunk 3 above)
Comment (doemser) [reply src/styles/global.css:19]: you resolved this, but it is not resolved

File: src/assets/fonts/inter.css [hunk 5]
```diff
@@ -0,0 +1,39 @@
+/* inter-regular - latin */
+@font-face {
```
Comment (doemser) [inline src/assets/fonts/inter.css:2]: instead of using this approach, you should take a look into next/fonts

Comment (ChristianMoschke) [reply src/assets/fonts/inter.css:2]: Is there any benefit from using "next/fonts"?

Comment (evabergold) [reply src/assets/fonts/inter.css:2]: next/fonts will automatically optimize your fonts (including custom fonts) and remove external network requests for improved privacy and performance by self-hosting Google fonts. (see handout react styled components 1)

**Review by evabergold — CHANGES_REQUESTED** [review]
I just tried to answer your question about next/fonts, otherwise it looks good :-)

**Pull request merged**